src/Components/Endpoints/src/RazorComponentEndpointInvoker.cs

```diff
@@ -32,24 +34,16 @@ private async Task RenderComponentCore()
     {
         _context.Response.ContentType = RazorComponentResultExecutor.DefaultContentType;
 
-        // We could pool these dictionary instances if we wanted, and possibly even the ParameterView
-        // backing buffers could come from a pool like they do during rendering.
-        var hostParameters = ParameterView.FromDictionary(new Dictionary<string, object?>
-                {
-                    { nameof(RazorComponentEndpointHost.RenderMode), RenderMode.Static },
-                    { nameof(RazorComponentEndpointHost.ComponentType), _componentType },
-                    { nameof(RazorComponentEndpointHost.ComponentParameters), null },
-                });
```

Comment:
This is cool! Does it mean we can delete `RazorComponentEndpointHost` entirely now? Or is it still required in some other case?

Reply:
I'm not sure. @javiercn?

Reply:
Oh right yes, that sounds correct.